src/Model/Term.php

```diff
@@ -176,7 +86,7 @@ public static function tagTextFromItemId($item_id)
 	 * @return array
 	 * @throws \Exception
 	 */
-	public static function tagArrayFromItemId($item_id, $type = [self::HASHTAG, self::MENTION])
+	private static function tagArrayFromItemId($item_id, $type = [self::HASHTAG, self::MENTION])
```

Comment:
Please rename this method to `getByItemId`
```suggestion
	private static function getByItemId($item_id, $type = [self::HASHTAG, self::MENTION])
```

Reply:
This function will be removed in one of the upcoming PRs anyhow :-)